src/styles/components/_fileinfo.scss

```diff
@@ -37,17 +37,14 @@
 
   .file-info-bg {
     bottom: 0;
+    color: $fileinfo-icon-background-color;
+    font-size: 20em;
     left: 41px;
     overflow: hidden;
     position: absolute;
     right: 0;
-    top: -17px;
+    top: -82px;
```

Comment:
This corrects alignment of the background icon inside the `.file-info` box because the `font-size` rule was moved here.